apps/mitt-konto/src/Components/Paywall.purs

```diff
@@ -0,0 +1,217 @@
+module MittKonto.Components.Paywall where
+
+import Prelude
+
+import Data.Array ((:), filter, findIndex, head, modifyAt, null)
+import Data.Int (fromString)
+import Data.Maybe (Maybe(..), fromMaybe, maybe)
+import Effect (Effect)
+import Effect.Aff as Aff
+import Effect.Class (liftEffect)
+import Effect.Exception (error)
+import Effect.Uncurried (mkEffectFn1)
```

Comment:
Using `mkEffectFn1` feels a bit too low level to me, I think I'd prefer using `handler_` and friends from `React.Basic.Events`.